lib/workers/repository/init/index.ts

```diff
@@ -32,9 +32,12 @@ export async function initRepo(
 ): Promise<RenovateConfig> {
   PackageFiles.clear();
   let config: RenovateConfig = initializeConfig(config_);
+
   await resetCaches();
-  config = await initApis(config);
-  await initializeCaches(config);
+  const workerPlatformConfig = await initApis(config);
+  config = workerPlatformConfig;
```

Comment:
Why these changes?

Reply:
`RenovateConfig` interface doesn't contain required fields, but in fact they're present after `initApis` call.

Reply:
I don't understand why the intermediate variable is needed though

Reply:
I wanted to avoid type assertion